### PR TITLE
Fix #145, fix #152, fix #153

### LINKED ISSUE #145: Consider making rich pictures less blurry

### LINKED ISSUE #152: Fix images in data visualization

### LINKED ISSUE #153: Separate Introduction & Analysis and Discussion & Conclusion

### PR DESCRIPTION
**Pull request by Danar435**

### CODE REVIEW

**Review by SebastianMygind — DISMISSED** [review]

File: src/discussion/final-thoughts.tex [hunk 1]
```diff
@@ -1,4 +1,4 @@
-\subsection{Final thoughts}\label{subsec:final-thoughts}
+\section{Final thoughts}\label{sec:final-thoughts}
```
Comment (MadsHeilmann) [inline src/discussion/final-thoughts.tex:1]: If we make subsections into sections we need to also change the folder structure. Each section should have its own folder with subsequent index.tex file

File: src/discussion/future-development.tex [hunk 2]
```diff
@@ -1,5 +1,5 @@
 % textidote: ignore begin
-\subsection{Future Development}\label{subsec:future-development}
+\section{Future Development}\label{sec:future-development}
```
Comment (MadsHeilmann) [inline src/discussion/future-development.tex]: Same as above

File: src/discussion/index.tex [hunk 3]
```diff
@@ -1,8 +1,6 @@
-% textidote: ignore begin
-\section{Discussion}\label{sec:discussion-conclusion}
-% textidote: ignore end
-As stated above, this section will discuss the features of the system and possible future developments and
-evaluate the software requirements that were set at the start of the project.
+In this chapter, we will discuss the features of the system and evaluate the software requirements that were set the
+start of the project.
+Then we will discuss the potential future development of the system.
```
Comment (MadsHeilmann) [inline src/discussion/index.tex]: This should be in a file in the "openings" folder. Not here

File: src/discussion/software-requirements-evaluation.tex [hunk 4]
```diff
@@ -1,5 +1,5 @@
 % textidote: ignore begin
-\subsection{Software requirements evaluation}\label{subsec:software-requirements-evaluation}
+\section{Software requirements evaluation}\label{sec:software-requirements-evaluation}
```
Comment (MadsHeilmann) [inline src/discussion/software-requirements-evaluation.tex]: Same as above

File: src/introduction/nova.tex [hunk 5]
```diff
@@ -1,5 +1,5 @@
 % textidote: ignore begin
-\subsection{NOVA}\label{subsec:nova}
+\section{NOVA}\label{sec:nova}
```
Comment (MadsHeilmann) [inline src/introduction/nova.tex]: Same as above

**Review by MadsHeilmann — CHANGES_REQUESTED** [review]

File: src/discussion/final-thoughts.tex (hunk 1 above)
Comment (Danar435) [reply src/discussion/final-thoughts.tex:1]: Ahh sorry about that, still not used to the new structure. Will fix when I get back home

Comment (Danar435) [reply src/discussion/final-thoughts.tex:1]: @MadsHeilmann what should we do with the conclusion? Keep it as is or move it to openings? It's a chapter without sections

**Review by MadsHeilmann — APPROVED** [review]

**Review by TexWO — APPROVED** [review]

**Pull request merged**